src/lib/generateSchemaJson.ts

```
@@ -1,12 +1,21 @@
 export function generateSchemaJson(schema: any, useExample = false) {
-  return formatJson(propertiesTypesJsonRecursive(schema, useExample))
+  return formatJson(propertiesTypesJsonRecursive(schema, useExample, new Set()))
 }
 
-export function propertiesTypesJsonRecursive(schema: any, useExample = false) {
+export function propertiesTypesJsonRecursive(schema: any, useExample = false, visited = new Set(), level = 0) {
+  if (visited.has(schema)) {
+    if (level > 10) {
+      return '[Circular Reference]'
+    }
+    visited = new Set()
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)